api/v1/services/TrackingService.js

```diff
@@ -0,0 +1,91 @@
+var CheckitError = require('checkit').Error;
+var _ = require('lodash');
+var _Promise = require('bluebird');
+var ms = require('ms');
+
+var cache = require('../../cache').instance();
+var TrackingItem = require('../models/UniversalTrackingItem');
+var errors = require('../errors');
+var utils = require('../utils');
+
+const TrackingFooter = "UniversalTracking";
```

Comment:
Call it `TrackingNamespace`. And make it `utracking_` so that it's pre-pended to the key, not appended.